util/buildflags/attests.go

```diff
@@ -10,6 +12,167 @@ import (
 	"github.com/tonistiigi/go-csvvalue"
 )
 
+type Attests []*Attest
```

Comment:
These types with `ToPB/String/(Un)Marshal*/FromToCty` are good candidates for unit tests. Some part of the helpers can probably be shared. I'm worried about cases where somebody makes a change in one of these functions but forgets to make a matching change in another.